internal/provider/atlas_schema_data_source_test.go

```diff
@@ -51,6 +63,9 @@ func TestAccSchemaDataSource(t *testing.T) {
 			{
 				Config: fmt.Sprintf(`data "atlas_schema" "market" {
 					dev_db_url = "mysql://root:pass@localhost:3307/test"
+					variables = [
```

Comment:
can't we use a map?
```hcl
variables = {
  tenant = "test"
}
```

Reply:
I think, this array of string will allow the user input array for a variable: `foo=item1`, `foo=item2`. But a map can't.